.travis.yml

```diff
@@ -1,28 +1,25 @@
 sudo: required
 language: node_js
 node_js:
-#  - stable
+  - 11
   - 10
-  - 9
   - 8
-#  - 6
-#  - 4
-#  - 0.12
-#  - 0.11
+cache: npm
```

Comment:
Are we utilizing cache if we use `npm ci`? I think it removes `.\node_modules` at the beginning.

Reply:
See the docs: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#npm-ci-support
`$HOME/.npm` is cached, so yes, this does help.

Reply:
Oh, forgot about that, somebody mentioned it before somewhere